common/protos/index.ts

```diff
@@ -15,21 +15,31 @@ export interface IProtoClass<IProto, Proto> {
 }
 
 // ProtobufJS's native verify method does not check that only defined fields are present.
-// TODO(ekrekr): swap to Typescript protobuf library rather than having to do this.
+// TODO(ekrekr): swap to Typescript protobuf library rather than having to do this; however using TS
+// libraries currently available would incur a significant performance hit b/305915462#comment18.
```

Comment:
Is there an external bug you can point to or replicate or else can you add this as a github issue?

Reply:
I will just remove the specific reference.